src/utils/constants.ts

```diff
@@ -28,7 +28,7 @@ export const API_URL = process.env.NEXT_PUBLIC_API_URL!;
 export const TOKEN_COOKIE = "token";
 export const REDIRECT_TO_COOKIE = "redirectTo";
 
-export const DOCS_URL = "https://docs.lurkr.gg";
+export const DOCS_URL = "https://lurkr.gg/docs";
```

Comment:
Remove this variable. Inline the links using just `/docs/`, and make sure to use next/link instead of ExternalLink.